build/gen_stub.php

```diff
@@ -3636,14 +3679,26 @@ function (FuncInfo $funcInfo) use($fileInfo, &$generatedFunctionDeclarations) {
     }
 
     if ($fileInfo->generateClassEntries) {
-        if (!empty($fileInfo->constInfos)) {
-            $code .= "\nstatic void register_{$stubFilenameWithoutExtension}_consts(int module_number)\n";
+        $classEntriesWithSensitiveParams = [];
+        $attributeInitializationCode = generateAttributeInitialization($fileInfo, $classEntriesWithSensitiveParams);
+
+        if ($attributeInitializationCode !== "" || !empty($fileInfo->constInfos)) {
+            $code .= "\nstatic void register_{$stubFilenameWithoutExtension}_symbols(int module_number";
+            foreach ($classEntriesWithSensitiveParams as $ce) {
+                $code .= ", zend_class_entry *$ce";
+            }
```

Comment:
This currently created duplicates if multiple methods of a single class have sensitive parameters (Example: `ZipArchive::setEncryptionName()` and `ZipArchive::setPassword()`).

Reply:
Thanks for testing, I've just addressed all the review comments

Reply:
Thanks. I've rebased #8352 onto your branch and pushed. Let's see how that goes :smiley: